Source/Core/EllipsoidGeodesic.js

```diff
@@ -171,14 +171,18 @@ define([
         ellipsoidGeodesic._uSquared = uSquared;
     }
 
+    //>>includeStart('debug', pragmas.debug);
```

Comment:
I wonder about these scratch variables:
Why do these scratch variables live outside the only scope they're used?
Why use scratch variables like this?

Reply:
While debugging (the web worker) I ran into a case where the "subdivide heights" scratch array had a zero-length even though the code that was being debugged had presumably just initialized it... I reproduced it once after a refresh (left the debugger open).  I've noticed strange behavior while debugging web workers with Chrome, so it's possible it's an artifact of that and not an issue with the logic (shared scratch) and some kind of race condition (between worker/worker or worker/main -- ... apologies I'm not well versed here).

Reply:
The scratch variables are at this scope so they can be reused each time the function is called instead of creating new instances each time.  This is something we do commonly for performance.  See https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#result-parameters-and-scratch-variables

So you'll need to remove the pragmas here

Reply:
Same here with adding the pragma block: these scratch variables are only used for the debug check.